zombies.js

```diff
@@ -138,6 +138,13 @@ let ak47 = new Weapon ("Ak47", 10);
  * @param {Item/Weapon/Food} item   The item to take.
  * @return {boolean} true/false     Whether player was able to store item in pack.
  */
+ takeItem(item) {
+    if(this._pack.length < 3) {
+      this._pack.push(item);
+      console.log(this.name + "" + item.name);
+   }   return true;
```

Comment:
Let's move this `return` statement down to the next line and push the closing `}` so it lines up with `if` (indentation)